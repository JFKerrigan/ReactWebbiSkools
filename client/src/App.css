.App {
  text-align: center;
}

body {
  background: hsl(199, 39%, 89%);
  text-align: center;
  font-family:'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
}

.container {
  /* display:flex; */
  flex-direction:column;
  justify-content:center;
}

.input {
  display: flex;
  justify-content: start; 
}

.available {
  display: flex;
  flex-direction: column;
}

.container input {
  width: 50%;
}

.label {
  margin-top: 10px;
}

.button{
  border-radius: 8%;
  width: 51%;
  height: 70px;
  justify-content: center;
  margin: 20px;
  background-color: lightyellow;
  font-family:'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif
}

input[type=text] {
  border-radius: 5%;
  margin: 5px;
  height: 20px;
}

.listQ {
  display: flex;
  flex-direction: column;
  justify-content: center;
  width: 200px;
  margin-top: 20px;
}


.answers {
  display: flex;
  flex-direction: column;
  justify-content: start;
  width:200px; 
  margin-top:20px;
}

.allDetails {
  justify-content: center;
  width: 80%;
  
}

.quizList {
  display: flex;
  flex-direction: column;
  justify-content: center;
}

.quizListCont {
  display: flex;
  justify-content: center;
}

.quizlist ol {
  justify-content: center;
  margin-left: 100px;
}

.nav {
  background-color: rgb(42, 120, 140);
  display: flex;
  justify-content: space-evenly;
  height: 50px
}

.test {
  font-size:x-large;
  justify-content: start;
}

.wrapper {
  display: flex;

}

.quizButton {
  display: flex;
  justify-content: center;
  border-radius: 8%;
  width: 51%;
  height: 70px;
  justify-content: center;
  margin: 20px;
  background-color: lightyellow;
  font-family:'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif
}

.buttonContainer  {
  display: flex;
  justify-content: center;
}