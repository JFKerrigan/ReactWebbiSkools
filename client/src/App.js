import React from 'react'
import { BrowserRouter, Routes, Route } from 'react-router-dom';
import Login from './components/Login';
import Register from './components/Register';
import Dashboard from './components/Dashboard';
import Quizzes from './components/Quizzes';
import Create from './components/Create';
import Logout from './components/Logout';

const App = () => {
	return (
		<div>
			
			<BrowserRouter>
			<Logout />
        		<Routes>
				  <Route exact path="/login" element={<Login />} />
				  <Route exact path="/register" element={<Register />} />
				  <Route exact path="/dashboard" element={<Dashboard />} />
				  <Route exact path='/quizzes' element={<Quizzes />} />
				  <Route exact path='/create' element={<Create />} />
        		</Routes>
			</BrowserRouter>
		</div>
	)
}

export default App