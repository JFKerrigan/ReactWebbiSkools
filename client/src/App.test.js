import { render, fireEvent,  screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import * as React from 'react'
import App from './App';

// const { axe, toHaveNoViolations } = require('jest-axe')
// expect.extend(toHaveNoViolations)

// it('should demonstrate this matcher`s usage with react', async () => {
//   render(<App/>, document.body)
//   const results = await axe(document.body)
//   expect(results).toHaveNoViolations()
// })

it('renders correctly', () =>{
  const text = renderer
    .create(<App />)
    .toJSON();
  expect(text).toMatchSnapshot()
})

