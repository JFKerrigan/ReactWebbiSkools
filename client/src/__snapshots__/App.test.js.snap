// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="nav"
  >
    <a
      href="/dashboard"
      onClick={[Function]}
    >
      Dashboard
    </a>
    <div>
      <a
        href="/login"
        onClick={[Function]}
      >
        Logout
      </a>
    </div>
  </div>
  <div
    class="container"
  >
    <h1>
      Welcome to WebbiSkools
    </h1>
    <h3>
      Please login to access your quizzes
    </h3>
    <button
      class="button"
      data-testid="homeButton"
    >
      <a
        href="/login"
        onClick={[Function]}
      >
        Login
      </a>
    </button>
  </div>
</div>
`;
